consumer/pdata/common.go

```diff
@@ -88,17 +88,17 @@ type AttributeValue struct {
 	//
 	// orig cannot be nil (it is an invalid instance if orig is nil).
 	// (*orig) may be nil.
```

Comment:
>(*orig) may be nil.

This comment is no longer applicable.

Reply:
This comment is longer applicable:
```
// Double pointer is required so that in the Setters we can update the pointer if
	// it is nil. This double pointer can be eliminated in the future if we embed AnyValue
	// using Gogoproto `(gogoproto.nullable) = false` annotation.
```

Reply:
done.